.pipelines/v2/templates/job-fuzz.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  - name: configuration
+    type: string
+    default: "Release"
+  - name: platform
+    type: string
+    default: ""
+  - name: inputArtifactStem
+    type: string
+    default: ""
+
+jobs:
+  - job: OneFuzz
+    pool:
+      vmImage: windows-2022
+
+    steps:
+      - checkout: self
+        submodules: false
+        clean: true
+        fetchDepth: 1
+        fetchTags: false
+
+      - download: current
+        displayName: Download artifacts
+        artifact: build-${{ parameters.platform }}-${{ parameters.configuration }}${{ parameters.inputArtifactStem }}
```

Comment:
If you create a variable for this you can clean it up on line 4 as well.

```yaml
variables:
  ArtifactName: build-${{ parameters.platform }}-${{ parameters.configuration }}${{ parameters.inputArtifactStem }}
```

```yaml
- download: current
  ...
  artifact: $(ArtifactName)
```

```yaml
onefuzzDropDirectory: $(Pipeline.Workspace)\$(ArtifactName)\x64...
```

Reply:
Thanks, I have updated the code in the latest commit.